src/py/mat3ra/notebooks_utils/ipython/_collab.py

```
@@ -5,15 +5,14 @@
 def get_notebook_info() -> dict:
     """
     Get the information about a currently running notebook in Google Colab.
-    Args:
-        None
+
     Return:
         a dict with notebook info.
     """
-    # ip = socket.gethostbyname(socket.gethostname())  # 172.28.0.12
     ip = os.getenv("COLAB_JUPYTER_IP")
     response = requests.get(f"http://{ip}:9000/api/sessions").json()[0]
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)